snomed/com.b2international.snowowl.validation.snomed/src/main/resources/scripts/rule664.groovy

```diff
@@ -92,12 +92,13 @@ if (params.isUnpublishedOnly) {
 		.each { Hits<String[]> publishedTermsBatch ->
 			// all returned terms are duplicate of an unpublished term
 			publishedTermsBatch.each { publishedTerm ->
-				if (activeConceptIds.get().contains(publishedTerm[1])) {
-					issues.add(ComponentIdentifier.of(SnomedDescription.TYPE, publishedTerm[0]))
-				}
+				String term = publishedTerm[0];
```

Comment:
Now `publishedTerm[1]` is completely unused, and it is not checked whether this (active) description is sitting on an active concept or not.

Reply:
True...